dockstore-webservice/src/test/resources/importTesting.wdl

```diff
@@ -0,0 +1,43 @@
+version 1.0
+
+import "./md5sum.wdl" as importedMapTask
+import "https://raw.githubusercontent.com/DockstoreTestUser2/wdl-1.0-workflow/master/hello.wdl" as importedHttpTask
```

Comment:
Import a specific tag/commit to avoid unexpected future breakage.
Better yet, a local file?

Reply:
This is specifically to test remote imports. But I agree, I'll use a commit id.